### ISSUE TITLE
Batch normalization

### ISSUE THREAD

**Issue by delhomer** [Oslandia/mapillary_classification#4]
As a common technique to improve the network performances (measured by accuracy, for instance), the batch normalization encoutered a huge success in neural network literature. See the paper of [Szegedy and Ioffe (2012)](https://arxiv.org/abs/1502.03167) for more details and [TensorFlow documentation](https://www.tensorflow.org/api_docs/python/tf/nn/batch_normalization) to have some clues on how to implement it within the project.

**Comment by delhomer** [comment]
Very clear and ready-to-use code in the dedicated [github repo](https://github.com/udacity/deep-learning/tree/master/batch-norm) provided by Udacity. Accurate modifications in the code under test phase.
